lib/internal/Magento/Framework/Url.php

```diff
@@ -1008,6 +1014,7 @@ public function getRebuiltUrl($url)
      * @param string $value
      * @return string
      * @deprecated 101.0.0
+     * @see https://github.com/magento-commerce/magento2ce/commit/6729b6e01368248abc33300208eb292c95050203
```

Comment:
@glo24157: no idea if this url is the best way to explain the deprecation, but you should at least link to a publicly available url, so this one is better `https://github.com/magento/magento2/commit/6729b6e01368248abc33300208eb292c95050203`

Linking to private repositories is definitely not helpful.